test/results/tiktok.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+from gallery_dl.extractor import tiktok
+
+PATTERN = r"https://p1[69]-.*\.tiktokcdn.*\.com/.*/[0-9a-fA-F]+~.*\.jpeg"
+
+
+__tests__ = (
+# Test many photos.
```

Comment:
Use `"#comment": "..."` after the initial `"#url": "..."`
https://github.com/mikf/gallery-dl/blob/454f766f5ec249de93317610d375e84899f7b35c/test/results/pixiv.py#L64-L66